core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -120,6 +120,8 @@ $(call force,CFG_DT,y)
 CFG_DTB_MAX_SIZE ?= 0x100000
 CFG_CORE_ASYNC_NOTIF ?= y
 CFG_CORE_ASYNC_NOTIF_GIC_INTID ?= 219
+CFG_CORE_ITR_NOTIF ?= y
```

Comment:
Better to test this on QEMU v8.

Reply:
ok, i'll make this be tested on both qemu_virt and qemu_armv8a.